wordpress-mobile/WordPress-FluxC-Android
# Add order shipping line fields

This will fix #1885 and will initially be used for enabling shipping refund in Woo Mobile.

**To test:**
1. Run the test on `example/src/test/java/org/wordpress/android/fluxc/wc/order/WCOrderModelTest.kt`
2. Make sure everything passes, especially `testGetShippingLines` and `testGetShippingLinesAttributes`
